cmd/tegola/example-configs/osm.toml

```diff
@@ -0,0 +1,356 @@
+[webserver]
+port = ":8080"
+
+[[providers]]
+name = "osm"
+type = "postgis"
+host = ""
+port = 5432
+database = "osm"
+user = "ARolek"
```

Comment:
This should be "". There are a few places with this; and others where it's "". I think "" is better for example config file, or something like "[USERNAME]" as "ARolek" is confusing.